frontends/common/options.cpp

```diff
@@ -76,12 +76,14 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
                        return true; },
                     "[Deprecated; use --std instead] Specify language version to compile.",
                     OptionFlags::Hide);
-    registerOption("--std", "{p4-14|p4-16}",
+    registerOption("--std", "{p4-14|p4-16|p4-18}",
```

Comment:
I'm aware that this is still work in progress, but as far as I know `14` and `16` was meant to represent the year in which the P4 language specification was developed: https://youtu.be/GslseT4hY1w?t=651

Is there going to be P4_18 language specification?

Reply:
I am not aware of any p4-18 specification on the horizon.  The p4-18 flag was a convenience to tell p4c that a P4 program is using new `override` and `super` keywords and also tell p4c to generate merged output p4-16 program.   I updated README.md to show how `ptest` is used.

Later, I can add an explicit flag or do something else.

Regarding code review, the few tests failing with this code also fail for me with a p4c without these changes.  So, this PR is not an issue with `make check`.  We can get a head start on code review.  The guts of the changes are in `parserDriver.h` and `p4parser.ypp`. thanks

Reply:
> Is there going to be P4_18 language specification?

Certainly not. Not even a P4_20. So far we have only incrementally extended P4_16 remaining backwards-compatible, so there was no need for changing the name of the language. I personally hope that we can stick with P4_16 for a long time.

Reply:
Mihai and I haven't colluded, I swear, but I'm also not a big fan of the years numbering. I think we'll stick with semantic versioning for a while...